### ISSUE TITLE
[QST] Multiple columns Apply

### ISSUE THREAD

**Issue by wahyubram** [rapidsai/cudf#10392]
I'm new in cudf, want asking about apply from multi columns apply in cudf.

with pandas we can do this:

```
def func_colC(col_A,col_B):
   p = Popen(['curl','https//www.example.com/q=' + col_A' + '&' + 'col_B'],stdout=PIPE,stderr=PIPE,shell=False)
   out,err = p.communicate()
   c = out.decode('utf-8')
   return c


df['col C'] = df[['colA', 'colB']].apply(lambda x : func_colC(*x),axis=1)
```

How to do this in cudf?

**Comment by brandon-b-miller** [comment]
In general, multicolumn `apply` in cuDF works the same way as in pandas, with some limitations. Something like this:

```python
def func(row):
    col_A = row['col_A']
    col_B = row['col_B']
    return col_A + col_B

df['col_C'] = df.apply(func, axis=1)
```

However the code inside `func` is currently limited to what a GPU thread can do, so some of the things you are attempting to write in your function are not supported such as use of `Popen` to load data in parallel. In addition, string functions are not currently supported such as `decode`, although hopefully these are coming soon.

For a guide on what's supported for UDFs, I recommend [this part](https://docs.rapids.ai/api/cudf/stable/user_guide/guide-to-udfs.html) of the docs.

**Comment by beckernick** [comment]
Closing this as answered and out of scope. Please feel free to re-open to continue the discussion.
